### PR TITLE
Added some E5 board + Fixes

### PR DESCRIPTION
**Pull request by hallard**
- Fixed console port on RAK3172
- fixed misnamed `RAK7231_radio_driver` to `RAK3172_radio_driver`
- fixed button 1 mapping

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,6 +91,27 @@ Use LoRa E5 module and added
...
 - Red Led on PB5 (LED2) reversed (1=OFF, 0=ON)
 
+LoRa-E5 use only RFO_HP so you need to add this to your `mbed_app.json` on section `target_overrides`
+
+```json
+
+   "LORA_E5": {
+      "stm32wl-lora-driver.rf_switch_config": 2
+   }
+```
```
Comment (jeromecoutant) [inline README.md]: Move this to "Seeed Studio LoRa E5" ?

File: README.md [hunk 2]
```diff
@@ -99,16 +120,41 @@ https://docs.rakwireless.com/Product-Categories/WisDuo/RAK3172-Module/Datasheet/
...
 
+RAK3172 use only RFO_HP, and no TXCO so you need to add this to your `mbed_app.json` on section `target_overrides`
+
+
+```json
+
+   "RAK3172": {
+      "stm32wl-lora-driver.rf_switch_config": 2,
+      "stm32wl-lora-driver.crystal_select" : 0
+   }
```
Comment (jeromecoutant) [inline README.md]: Same comment, move this part above ?

File: TARGET_STM32WL/CMakeLists.txt [hunk 3]
```diff
@@ -3,5 +3,7 @@
 
 add_subdirectory(TARGET_LORA_E5 EXCLUDE_FROM_ALL)
 add_subdirectory(TARGET_LORA_E5_BREAKOUT EXCLUDE_FROM_ALL)
+add_subdirectory(TARGET_LORA_E5_MINI EXCLUDE_FROM_ALL)
+add_subdirectory(TARGET_LORA_E5_DEV_BOARD EXCLUDE_FROM_ALL)
```
Comment (jeromecoutant) [inline TARGET_STM32WL/CMakeLists.txt:7]: These new targets are not in the readme

File: README.md (hunk 2 above)
Comment (hallard) [reply README.md]: All done :-)

**Comment by hallard** [comment]
All requested changes done

**Comment by jeromecoutant** [comment]
Mmm,I know you love git commands...
But, same rule as in mbed-os repo, merge commit  is not allowed.
Could you rebase your commits on top of master, and force push again to this PR ? Thx

File: README.md [hunk 4]
```diff
@@ -135,6 +139,38 @@ TARGET: LORA_E5
...
+
+   "LORA_E5": {
+      "stm32wl-lora-driver.rf_switch_config": 2
+   }
+```
+
+## Seeed Studio LoRa E5 Mini
+
+Use LoRa E5 modulen basic board with USB/Serial and headers
+
```
Comment (jeromecoutant) [inline README.md:162]: As you can see for other  targets, I added, for each one, some information to match this doc file with the json file.
Which target should I build for this target ?

Thx
```
TARGET: XXX
```

See https://github.com/ARMmbed/stm32customtargets#manual-pull-request-checks

Comment (hallard) [reply README.md:162]: yep I updated readme with:

- `LORA_E5_DEV_BOARD`
- `LORA_E5_MINI`

**Comment by jeromecoutant** [comment]
I am sorry, last commit is a merge one...
Could you rebase your commits on top of master, and force push again to this PR ? 
(clean branch starting from origin/master)
Thx

**Review by jeromecoutant — CHANGES_REQUESTED** [review]
Waiting for a clean branch

**Comment by hallard** [comment]
> I am sorry, last commit is a merge one...
> Could you rebase your commits on top of master, and force push again to this PR ?
> (clean branch starting from origin/master)
> Thx

humm, I'm very sorry sometimes git is equal to some magic for me. I merged upstream/master (your repo) into my master repos (the one I forked from you), then I also merged `upstream/master` into this branch  `E5_DevBoards`which contains these changes. so for me it's up to date with master and contains only changes, but my be I'm missing something.

It looks like it's confirmed by git client, rebase from `origin/master` is not possible because it's already up to date

image

and from `upstream/master` either
image

Do I miss something? I can also close this PR, save my files, create new clean branch/new PR with saved file if you're more confortable with that. Just let me know.

**Comment by hallard** [comment]
replaced by clean #15

**Pull request closed without merging**